draft-ietf-quic-manageability.md

```diff
@@ -611,7 +609,9 @@ re-establishment, these can be ignored for RTT measurement purposes.
 
 Handshake RTT can be measured by adding the client-to-observer and
 observer-to-server RTT components together. This measurement necessarily
-includes any transport and application layer delay at both sides.
+includes any transport and application layer delay (the latter mainly
+ascribable to the asymmetric crypto operations associated with the TLS
```

Comment:
"ascribable" is possibly confusing to non-native speakers; suggest simplification.

```suggestion
caused by the asymmetric crypto operations associated with the TLS
```